xds/src/main/java/io/grpc/xds/XdsAttributes.java

```diff
@@ -32,5 +34,21 @@
   public static final Attributes.Key<SdsSecretConfig> ATTR_SDS_CONFIG =
           Attributes.Key.create("io.grpc.xds.XdsAttributes.sdsSecretConfig");
 
+  /**
+   * Attribute key for TlsCertificate of a subchannel.
+   *
```

Comment:
nit: those extra lines bother me. can you remove those (L39, L47, L27?)

Reply:
done